src/QueryPipeline/QueryPipeline.cpp

```diff
@@ -26,7 +26,6 @@
 #include <Processors/Transforms/StreamInQueryCacheTransform.h>
 #include <Processors/Transforms/TotalsHavingTransform.h>
 #include <QueryPipeline/Chain.h>
-#include <QueryPipeline/Pipe.h>
```

Comment:
Pipe.h is clearly being used in this file. Whichever linter you are using is wrong